CONTRIBUTING.md

```diff
@@ -6,6 +6,25 @@ Please follow common guidelines for our projects [here](https://github.com/packi
 
 - [List of known issues](https://github.com/packit/packit-service-fedmsg/issues) and in case you need to create a new issue, you can do so [here](https://github.com/packit/packit-service-fedmsg/issues/new).
 
+## Testing
+
+We recommend running the tests in a container.
+
+In order to install the dependencies required to build the test image run:
+
+    make install-check-in-container-deps
+
+To build the test image run:
+
+    make build-test-image
+
+You can run the test now with:
+
+    make check-in-container
+
+The `install-check-deps` and `check` make-targets are also available if you
```

Comment:
@lachmanfrantisek @jpopelka I chose to mention these just like this.

The issue is that without installing the dependencies, just recommending `make install-check-deps && make check` will not work.

But I see no easy way to install the dependencies, and come up with a target or recommendation that would work for everyone.

Any ideas, or is this good enough?

Reply:
I don't understand what's the problem.

> without installing the dependencies, just recommending `make install-check-deps && make check` will not work.

Isn't `make install-check-deps` installing the dependencies? I'm lost.

Reply:
Installs only `pytest`. Yes, I know the name suggests otherwise. And would be great to install both test and project dependencies with it. I'm just not sure how to do this without keeping a list of dependencies at yet another place.

Testing in a container is much more straightforward from this POV.

So I'm looking for ideas on how to fix this. Although given that this is something that was recently introduced, I would be okay with dropping it.